dvc/dagascii.py

```diff
@@ -61,99 +63,14 @@ def __init__(self, cols, lines):
     def draw(self):
         """Draws ASCII canvas on the screen."""
         if sys.stdout.isatty():  # pragma: no cover
-            from asciimatics.screen import Screen
-
-            Screen.wrapper(self._do_draw)
+            lines = []
+            for line in self.canvas:
+                lines.append("".join(line))
+            pydoc.pager(os.linesep.join(lines))
         else:
```

Comment:
Seems like this `if isatty else` doesn't make sense here, as pydoc already handles it, no?

Reply:
good catch thanks and done